src/natcap/invest/sdr/sdr.py

```diff
@@ -566,14 +565,24 @@ def execute(args):
         target_path_list=[f_reg['sed_deposition_path'], f_reg['f_path']],
         task_name='sediment deposition')
 
-    _ = task_graph.add_task(
-        func=_calculate_sed_retention_index,
+    avoided_erosion_task = task_graph.add_task(
+        func=_calculate_avoided_erosion,
+        args=(
+            f_reg['rkls_path'], f_reg['usle_path'],
+            f_reg['avoided_erosion_path']),
+        dependent_task_list=[rkls_task, usle_task],
+        target_path_list=[f_reg['avoided_erosion_path']],
+        task_name='calculate local retention')
```

Comment:
Is this task name the inverse from the function call name? Would it make sense for the task name to reflect the function name at all? I feel like they are the same thing just said differently.

Reply:
Whoops, that's from a botched find-and-replace.  I think at one point we were calling this "local retention", but settled on "avoided erosion".  I added it in the task name. cca41e494